data/clr-boot-manager-mount-boot.service.in

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=mount /boot using clr-boot-manager
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=@BINDIR@/clr-boot-manager mount-boot
+ExecStop=@BINDIR@/umount /boot
```

Comment:
I think clr-boot-manager supports using a different boot directory than `/boot`, you should replace the `/boot` here with a template option.

Reply:
Goot catch, updated!